src/components/TitleContainer/TitleContainer.js

```diff
@@ -0,0 +1,27 @@
+import "./TitleContainer.css"
+
+function TitleContainer() {
+
+    const pathArray = window.location.pathname.split('/');
+    console.log(pathArray);
+
+    const title = "r/" + pathArray[2];
+
+    return (
+        <>
+            <div className = "header-title">
+
+                <h1>
+                    {title}
+                </h1>
+                <p>
+                    Subreddit specifications
+                </p>
+            </div>
+        </>
```

Comment:
De lege tags zijn niet nodig, omdat het maar één child bevat
```suggestion
        <div className = "header-title">
            <h1>
                {title}
            </h1>
            <p>
                Subreddit specifications
            </p>
        </div>
```